File: Frontend-CRM/src/features/sites/utils/irbCatalog.ts
/** IRB catalog: no client-side exclusions. */

export function normalizeIrbListName(name: string | null | undefined): string {
  return String(name ?? '')
    .trim()
    .toLowerCase()
    .split(/\s+/)
    .join(' ')
}

/** Hide from dropdowns; treat mappings to these as invalid. */
export function isIrbExcludedFromCatalog(
  _name: string | null | undefined,
  _uniqueCode?: string | null | undefined
): boolean {
  return false
}

File: Backend-CRM/app/modules/sites/irb_catalog.py
"""IRB catalog rules: all IRBs are allowed in pickers."""
from __future__ import annotations

from typing import Optional


def normalize_irb_list_name(name: Optional[str]) -> str:
    if not name:
        return ""
    return " ".join(str(name).strip().lower().split())


def is_irb_excluded_from_catalog(name: Optional[str], unique_code: Optional[str] = None) -> bool:
    """No IRBs are excluded from dropdowns or mapping."""
    return False
